### PR TITLE
A JavaScript clock.

### PR DESCRIPTION
**Pull request by surbhi400**
It is a simple clock that reads current time and is made using HTML,CSS and JS.

closes issue No #22  

Screenshot:
clock(2)

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+
+>>>>>>> b8662e061d9406bea9b70cf91f7e584f1c89477e
```
Comment (Yashika1410) [inline contributors.md:5]: please remove these head and commit numbers

File: contributors.md [hunk 2]
```diff
@@ -8,3 +24,8 @@
 * [Bagus](https://github.com/Bagusin)
 * [Bikas Kumar Behera](https://github.com/Bikaskumar572)
 
+<<<<<<< HEAD
```
Comment (Yashika1410) [inline contributors.md:27]: Resolve these conflicts and add your details in this contributors.md

**Review by Yashika1410 — CHANGES_REQUESTED** [review]
@surbhi400 Please resolve contributors.md conflicts and add your details

**Comment by surbhi400** [comment]
@Yashika1410 please review my PR again and let me know if any more changes is required.

**Review by Yashika1410 — APPROVED** [review]

**Pull request merged**